server/views/includes/index-poetry.pug

```diff
@@ -0,0 +1,139 @@
+                                | Вишни цвет расцвел в душе,
+                                br
+                                | Не смотря на снег.
+                                br
+                                | За окном уже весна
+                        .col-2
+                .carousel-item
+                    .row
+                        .col-2
+                        .col 
+                            p 
+                                | Темный небосвод.
+                                br
+                                | Уплывают облака.
+                                br
+                                | Зонтик от дождя
+                        .col 
+                            p 
+                                | Сумерки в душе.
+                                br
+                                | Бесконечный мыслей ход.
+                                br
+                                | Снова дождь
+                        .col 
+                            p
+                                | Души снегом скованы
+                                br
+                                | Где живет любовь?
+                                br
+                                | Мы забыли туда путь
+                        .col 
+                            p
+                                | Середина февраля.
+                                br
+                                | Рождение солнца
+                                br
+                                | С улыбкой на устах
+                        .col-2
+                .carousel-item
+                    .row 
+                        .col-2
+                        .col 
+                            p 
+                                | Не оглядывайся,
+                                br
+                                | Нить Ариадны исчезнет.
+                                br
+                                | Плачущая луна.
+                        .col
+                            p 
+                                | Подожди, я с тобою,
+                                br
+                                | Ветра дух и шелест листвы.
+                                br
+                                | Улетаю
+                        .col
+                            p 
+                                | Отпусти—отпускаю
+                                br
+                                | Дождь смывает последние слезы.
+                                br
+                                | Прости и прощай
+                        .col
+                            p 
+                                | Принятие…
+                                br
+                                | Подожди, я с тобой попрощаюсь.
+                                br
+                                | Эхо в пустоте
+                        .col-2
+
+            a(class="carousel-control-prev" href="#carouselExampleIndicators" role="button" data-slide="prev")
```

Comment:
I don't know whether it's just that I can't find the element with this ID - but where would this take you to? :)

Reply:
href in line nr 134 takes you to the previous slide of "#carouselExampleIndicators" (now changed to "carouselIndicators") - it returns you to the div in the top of the carousel element - line nr 3. It's a Bootstrap component with Javascript behind the scenes.